sources/functions/rutorrent

```diff
@@ -113,7 +113,7 @@ function rutorrent_install() {
 		"stat"	=> '',			// Something like /usr/bin/stat. If empty, will be found in PATH.
 	);
 	
-	$localHostedMode = false; 		// Set to true if rTorrent is hosted on the SAME machine as ruTorrent
+	$localHostedMode = true; 		// Set to true if rTorrent is hosted on the SAME machine as ruTorrent
```

Comment:
I think it would be nice to apply this to previous installations as well, any chance you could write an update function for this, please?

Reply:
`ruTorrent` does not support updating currently. The user is required to remove and reinstall. In the future, when updating support is added, these changes will apply by default.

Reply:
I'm referring to our own update files, namely https://github.com/swizzin/swizzin/blob/master/scripts/update/rutorrent.sh

This is where we put any config changes that need to be applied to old installs.

Reply:
This change is optional. It doesn't need to be applied to old installs. It will use the `XMLRPC` method if set to `false`. I don't think we should write an update function for this. It's more code to maintain, that will eventually need depreciated.